### DISCUSSION TITLE
add an account for testing.

### DISCUSSION THREAD

**Discussion by Rubiidev-IPad** [cytrinc/bark-coding#218] [General]
username: test1
password: testpass
this is for testing accounts

**Comment by raynepaws** [comment]
what does "testing accounts" mean?

**Comment by mariocraft987** [comment]
the password, you just revealed it lmao

**Reply by raynepaws** [reply]
yea i don't feel it's very smart to use that password since these convos are public

**Comment by dumorando** [comment]
deleting the account rn!
